### DISCUSSION TITLE
$? bash command doesn't show exit codes in Github Actions

### DISCUSSION THREAD

**Discussion by kabam-zmohamad** [community/community#60878] [Actions]
### Select Topic Area

Bug

### Body

Currently previous exit code ($?) doesn't show previously exited code. I've tested this with Python/Go/bash programs with different built-in exit libraries and it doesn't show up on the github actions runner.

It works on my local machine. The need for this functionality is to allow different ways of notifying and resolving different exit codes with third party libraries. 

Is this a bug with Github Actions? Or more of a security feature with it to reduce our access to the runners?

**Comment by piranna** [comment]
Related to https://github.com/orgs/community/discussions/46992
